README.md

```diff
@@ -1,4 +1,7 @@
 # Linkedin-quiz-questions
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
+<!-- ALL-CONTRIBUTORS-BADGE:END -->
```

Comment:
please remove it because this should be updated according to all contributors command 
more info could be found [here](https://allcontributors.org/docs/en/overview)